code/game/objects/structures/resindoor.dm

```diff
@@ -0,0 +1,118 @@
+/obj/structure/resindoor
```

Comment:
why didn't you just make this a subtype of mineral doors or firelocks?

Reply:
I tried that actually at the beginning but couldn't figure out how to make the door work with only aliens so i made a separate door with similar code but making it unique so only aliens can open and close it.